backend/app/tests/utils/inquiry.py

```diff
@@ -0,0 +1,19 @@
+import random
+import string
+
+from sqlmodel import Session
+
+from app import crud
+from app.models import Inquiry, InquiryCreate
+
+
+def create_random_inquiry(db: Session) -> Inquiry:
```

Comment:
good idea to move this to the utils folder but i have very similar logic here.  I'm not sure we need both
https://github.com/Incredihire/Survey/pull/44/files#diff-d78597f8d24b92f30cd93ec966dd134c745aa2b4499531296aec547583e456c5R15

Reply:
You don't need this at all if you mock out the create_inquiry funciton as suggested above

Reply:
Yes, I agree. I plan to revise my tests once we finalize testing tools.

Reply:
great